app/models/billing/contact.rb

```diff
@@ -20,7 +20,7 @@
 
 class Billing::Contact < Yeti::ActiveRecord
   self.table_name = 'notifications.contacts'
-  has_paper_trail class_name: 'AuditLogItem'
+  has_paper_trail class_name: 'AuditLogItem' if Rails.configuration.yeti_web['versioning_disable_for_models'].exclude?(name)
```

Comment:
could you incapsulate 
`has_paper_trail class_name: 'AuditLogItem' if Rails.configuration.yeti_web['versioning_disable_for_models'].exclude?(name)` in some module ?